doc/install_linux_easy.md

```diff
@@ -73,17 +73,17 @@ Which should display the same message the gcc call did, i.e.,
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
-### Step 3 - Clone Chi-Tech
+### Step 3 - Clone OpenSn
 
-**Important:**  If you want to contribute to **Chi-Tech**, it is strongly recommended to first fork the **Chi-Tech** repository into your own Git account and then to clone your fork. 
+**Important:**  If you want to contribute to **OpenSn**, it is strongly recommended to first fork the **Chi-Tech** repository into your own Git account and then to clone your fork. 
 
-Clone the **Chi-Tech** repository.  Go the folder where you want to keep Chi-Tech relevant stuff:
+Clone the **OpenSn** repository.  Go the folder where you want to keep Chi-Tech relevant stuff:
 ```bash
-    $ git clone https://github.com/chi-tech/chi-tech
+    $ git clone https://github.com/Open-Sn/opensn.git /path/to/opensn
```

Comment:
The previous sentence said "Go to folder", so no need to specify the path. But, may be even better is to provide the instructions like this:
```
$ cd /some/path/
$ git clone ...
```